tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -107,3 +107,23 @@ def test_allennlp_executor_with_include_package_arr() -> None:
         )
         result = executor.run()
         assert isinstance(result, float)
+
+
+def test_save_best_config() -> None:
+    with tempfile.TemporaryDirectory() as tmp_dir:
+
+        def objective(trial: optuna.Trial) -> float:
+            trial.suggest_uniform("DROPOUT", dropout, dropout)
+            executor = optuna.integration.AllenNLPExecutor(trial, config_file, tmp_dir)
+            return executor.run()
+
+        config_file = "tests/integration_tests/allennlp_tests/example.jsonnet"
+        dropout = 0.5
+
+        study = optuna.create_study(direction="maximize")
+        study.optimize(objective, n_trials=1)
+
+        best_config = optuna.integration.allennlp._save_best_config(config_file, study)
```

Comment:
Ditto. If you are okay with my suggestion...
```suggestion
        best_config = optuna.integration.allennlp._dump_best_config(config_file, study)
```